subprojects/detekt.yml

```diff
@@ -635,7 +635,7 @@ style:
         maxLineLength: 120
         excludePackageStatements: true
         excludeImportStatements: true
-        excludeCommentStatements: false
+        excludeCommentStatements: true
```

Comment:
Excluded comments.
It's needed from code blocks and diagrams in KDoc.
I don't want to suppress the whole method\class because of it.